docs/performance-budgets.md

```diff
@@ -0,0 +1,177 @@
+# Performance Budgets (budget.json)
+
+Use a performance budget to prevent site performance from regressing over time.
+
+## Usage
+
+Performance budgets are supported in the CLI version of lighthouse. To use:
+1. Create a `budget.json` file.
+2. When running Lighthouse from the command line, pass the `--budget-path` (or `--budgetPath`) flag followed by the path to budget file in order to calculate whenever a category is over budget.
+```
+lighthouse https://youtube.com --budget-path=budget.json
+```
+
+## budget.json
+The `budget.json` file is an array containing one or more `Budget` objects.
+
+```json
+[
+  {
+    "path": "/*",
+    "options": {
+       "firstPartyHostnames": ["*.my-site.com", "my-site.cdn.com"]
+    },
+    "timings": [
+      {
+        "metric": "interactive",
+        "budget": 5000
+      },
+      {
+        "metric": "first-meaningful-paint",
+        "budget": 2000
+      }
+    ],
+    "resourceSizes": [
+      {
+        "resourceType": "total",
+        "budget": 500
+      },
+      {
+        "resourceType": "script",
+        "budget": 150
+      }
+    ],
+    "resourceCounts": [
+      {
+        "resourceType": "third-party",
+        "budget": 100
+      }
+    ]
+  },
+  {
+    "options": {
+       "firstPartyHostnames": ["*.my-site.com", "my-site.cdn.com"]
+    },
+    "path": "/checkout",
+    "resourceSizes": [
+      {
+        "resourceType": "script",
+        "budget": 200
+      }
+    ]
+  }
+]
+``` 
+
+## Further Explanation
+
+### Timing Budgets
+
+_Lighthouse 6 & up_
+
+Use the optional `timings` property to define budgets for time-based performance metrics. In this context, budgets are defined in  milliseconds.
+
+```json
+"timings": [
+   {
+         "metric": "interactive",
+         "budget": 5000
+   }
+]
+```
+
+Supported timing metrics:
+
+- `first-contentful-paint`
```

Comment:
we should add `largest-contentful-paint` support to this list before 6.0 final :)

Reply:
Let's just add it now? + CLS. If you don't like that we could at least make an issue now.